app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable, :trackable
```

Comment:
@sinpai as we discussed before: no need for comments if it's not a hacky logic, which you are trying to explain other developers

Reply:
:+1: